pages/home.jsx

```diff
@@ -101,13 +105,15 @@ var BlogSection = React.createClass({
     }
   },
   componentDidMount: function() {
-    var self = this;
-    blogFeedLoader(this, window, "https://blog.webmaker.org/tag/teachtheweb/feed", function(data) {
-      self.setState({
-        featuredPostData: data.featuredPostData,
-        latestPostsData: data.latestPostsData
+    this.props.loadBlogPosts(function(data) {
+      if (!this.isMounted()) {
+        return;
+      }
+      this.setState({
+        featuredPostData: data.featuredPosts,
+        latestPostsData: data.latestPosts
```

Comment:
yea i agree it's redundant here, please get rid of them.  I do realize I tend to overuse `data` in variable names a lot :)